HHS/Head-Start-TTADP
# [TTAHUB-386] display new report status after submit

## Description of change
After form submit, a report is asynchronously updated in the database but some fields, like the new calculated status field seem to come back from the API incorrectly, probably as a result of additional asynchronous calculation. 

This code change leverages the location history passed through the form to ensure a newly submitted form displays correctly in the UI without waiting for the DB to update.

This PR also fixes an [accessibility issue](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-387) where a previous update introduced a bug into a commonly-used component.

## How to test

- Create activity report
- submit for approval
- look at the status of the newly created/submitted report's status under "my alerts" section
- refresh page, look at newly created/submitted report's status under "my alerts" section

For both steps 3 & 4, the newly submitted report should have a status of submitted (prior to this change, the status in step three for the newly submitted report would still say "draft")

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-386
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-387

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [n/a] Staging smoke test completed

### After merge/deploy

- [n/a] Update JIRA ticket status
